src/core/credentials/transport/tls/spiffe_utils.cc

```diff
@@ -18,16 +18,30 @@
 
 #include "src/core/credentials/transport/tls/spiffe_utils.h"
 
+#include <openssl/pem.h>
```

Comment:
Are these 3 openssl includes still needed?

Reply:
x509.h is needed for X509_free, but the others are not